### PR TITLE
Add title, author, and abstract to project metadata.

### LINKED ISSUE #613: Populate QGIS project-level metadata
Populate project-level metadata (e.g., `project_abstract`, `project_author`, `project_title`).

This information can be viewed via the Project properties view: `Project -> Properties...`, then the `Variables` tab.

Assuming we can use the PyQGIS library to set these attributes when we create the project file.

### PR DESCRIPTION
**Pull request by trey-stafford**
## Description

Title. Closes #613 .

## Checklist

If an item on this list is done _or not needed_, check it with `[x]` or click the
checkbox.

- [x] The PR description links to issues that it resolves with `closes #{issue_number}`
- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Environment lockfile updated if needed (`conda-lock`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build)`)
- [ ] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: qgreenland/util/qgis/project.py [hunk 1]
```diff
@@ -87,6 +89,22 @@ def make_qgis_project_file(path: Path) -> None:
     project.clear()
 
 
+def _add_project_metadata(project: qgc.QgsProject) -> None:
+    """Add project-level metadata (title, authors, abstract).
+
+    This information is viewable in the **Project > Properties** in QGIS.
+    """
+    project_metadata = project.metadata()
+    project_metadata.setAuthor("QGreenland team")
```
Comment (trey-stafford) [inline qgreenland/util/qgis/project.py:98]: Should we use our names, or is "QGreenland team" appropriate here?

File: qgreenland/util/qgis/project.py [hunk 2]
```diff
@@ -87,6 +89,22 @@ def make_qgis_project_file(path: Path) -> None:
...
+    project_metadata.setTitle("QGreenland")
+    project_metadata.setAbstract(
+        """QGreenland is a free and open-source Greenland-focused GIS package
+for QGIS. To learn more about QGreenland, visit our website:
+https://qgreenland.org"""
```
Comment (trey-stafford) [inline qgreenland/util/qgis/project.py:104]: How does this look? More detail? I think directing users to the website quickly is a good idea.

File: qgreenland/util/qgis/project.py (hunk 1 above)
Comment (MattF-NSIDC) [reply qgreenland/util/qgis/project.py:99]: I think `QGreenland team` is great. The full citation is easy to find. I think we should link to that page in our docs from this metadata.

File: qgreenland/util/qgis/project.py (hunk 2 above)
Comment (MattF-NSIDC) [reply qgreenland/util/qgis/project.py:104]: I agree. I think we could add one more paragraph here linking to the "how to cite" page, and we'd be golden.

**Review by MattF-NSIDC — APPROVED** [review]

File: qgreenland/util/qgis/project.py (hunk 2 above)
Comment (trey-stafford) [reply qgreenland/util/qgis/project.py:104]: Great idea, thanks!

**Pull request merged**